modules/s1/main.tf

```diff
@@ -0,0 +1,570 @@
+provider "aws" {
+  region = var.region
+}
+
+locals {
+  api_domain_name = join(".", [var.api_subdomain, var.domain_name])
+}
+
+### VPC
+module "vpc" {
+  source = "terraform-aws-modules/vpc/aws"
+
+  enable_ipv6 = true
+
+  name = join("-", [var.env_name, var.region, "vpc"])
+  cidr = "10.0.0.0/16"
+
+  azs             = ["eu-central-1a", "eu-central-1b", "eu-central-1c"]
```

Comment:
nit: make dependent on `var.region`

Reply:
What's your desired var.region value `eu-central` or `eu-central-1` ?

Reply:
We can just fetch them via a data source:
```hcl
provider "aws" {
  region = "eu-central-1"
}

data "aws_availability_zones" "this" {}

locals {
  azs = data.aws_availability_zones.this.names
}

output "azs" {
  value = local.azs
}

```